contribs/drt/src/main/java/org/matsim/contrib/drt/sharingfactor/SharingFactorModule.java

```diff
@@ -0,0 +1,38 @@
+package org.matsim.contrib.drt.sharingfactor;
+
+import org.matsim.api.core.v01.Id;
+import org.matsim.api.core.v01.population.Person;
+import org.matsim.contrib.drt.run.DrtConfigGroup;
+import org.matsim.contrib.dvrp.run.AbstractDvrpModeModule;
+import org.matsim.core.controler.MatsimServices;
+
+/**
+ * @author nkuehnel / MOIA
+ */
+public class SharingFactorModule extends AbstractDvrpModeModule {
+
+    private final DrtConfigGroup drtConfigGroup;
+
+    public SharingFactorModule(DrtConfigGroup drtConfigGroup) {
+        super(drtConfigGroup.getMode());
+        this.drtConfigGroup = drtConfigGroup;
+    }
+
+    @Override
+    public void install() {
+        bindModal(SharingFactorTracker.class).toProvider(modalProvider(getter ->
+                new SharingFactorTracker(new SharingFactorTracker.GroupPredicate() {
+                    @Override
+                    public boolean isGroupRepresentative(Id<Person> personId) {
+                        return SharingFactorTracker.GroupPredicate.super.isGroupRepresentative(personId);
+                    }
+                }))).asEagerSingleton();
```

Comment:
This is super complex, especially `SharingFactorTracker.GroupPredicate.super.isGroupRepresentative(personId)`.

I think it is equivalent to this:
```java
bindModal(SharingFactorTracker.class).toProvider(modalProvider(getter ->
                new SharingFactorTracker(personId -> true))).asEagerSingleton();
```